### ISSUE TITLE
Improve error message if no postscript viewer is installed on Windows

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#2114]
A issue reported by a Chinese user.
![image](https://user-images.githubusercontent.com/3974108/69384687-f42f9280-0c8a-11ea-98c2-3237ae7d4711.png)

The error message says "Windows can't find gsview64c. Please check if the filename is correct, then try again."

It's because "gmt end show" tries to open a PostScript file but there is no PostScript viewers installed. 

I think we need to check if gsview64c is available before calling it. BTW, since ghotscript is always available, can we call "gswin32c" or "gswin64c"?

**Comment by joa-quim** [comment]
Win installers for GMT6 come with gswin64c.exe

**Comment by joa-quim** [comment]
Sorry, missed the name. ``gsview64c.`` 
I'm afraid it's a bug. I see that ``docs.c`` has
```
ps_viewer = "cmd /c start gsview64c";	/* No previewer. Resort to this. */
```
but there is no such a thing as ``gsview64c``, only ``gswin64c``

**Comment by seisman** [comment]
Yes, but the error message says **gsview64c**

**Comment by joa-quim** [comment]
The workaround for him is to make a copy of ``gswin64c.exe`` and name it ``gsview64c.exe``

**Comment by seisman** [comment]
Closed by #2115
